policyengine_us/parameters/gov/ed/pell_grant/efc/head/income_modification.yaml

```
@@ -0,0 +1,13 @@
+description: The Education Depaurtment uses these values to ajust the income of the head and their spouse based on the formula being used.
```

Can you be more specific though? Do they multiply them?
```suggestion
description: The Education Department adjusts the income of the head and their spouse by these factors based on the formula being used.
```